habit-tracker/C11/asiu3209/project2/src/Pages/HomePage.tsx

```diff
@@ -0,0 +1,87 @@
+import NavBar from "../components/NavBar";
+import HabitForm from "../components/HabitForm";
+import HabitCard, { FormHabit, Habit } from "../components/HabitCard";
+import { useEffect } from "react";
+let nextID = 0;
+type habitStorageProp = {
```

Comment:
Types should be UpperCased, so `HabitStorageProp`, props should also be plural, so `HabitStorageProps`. Also we generally name the props after the component, so `HomePageProps`